benchmarks/dataframe.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python3
+import string
+import time, argparse
+import numpy as np
+import pandas as pd
+
+import arkouda as ak
+import random
+
+OPS = ['_repr_html_', '_get_head_tail_server', '_get_head_tail']
+TYPES = ('int64', 'uint64',)
+
+def generate_dataframe(N):
+    types = [ak.Categorical, ak.pdarray, ak.Strings, ak.SegArray]
+
+    # generate random columns to build dataframe
+    df_dict = {}
+    for x in range(20):  # loop to create 20 random columns
+        key = f"c_{x}"
+        d = types[random.randint(0, len(types)-1)]
```

Comment:
I'm thinking we should cycle deterministically through the types, rather than choosing them randomly. If today's run gets 10 Categorical columns and tomorrow's run gets 2, there will be a big difference in runtime that doesn't reflect anything about the code being benchmarked. Generating random *data* of a particular type is fine, though.

Reply:
I updated this to use `x%4` to evenly distribute the column types.